co2eq/meal.js

```diff
@@ -58,12 +60,14 @@ const CARBON_INTENSITY = { // kgCO2eq / kg
   // dairy
   'Cheese': 8.55, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
   'Milk': 1.29, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
+  'Milk (2%)': 1.9, // http://static.ewg.org/reports/2011/meateaters/pdf/methodology_ewg_meat_eaters_guide_to_health_and_climate_2011.pdf page 23
```

Comment:
I would rather not add this one as the other one (Milk) is a systemic review with multiple studies